k8s101.md

```diff
@@ -431,7 +431,7 @@ spec:
         run: my-nginx
     spec:
       containers:
-      - image: nginx:1.91 # <-- TYPO: version should be 1.9.1
+      - image: nginx:999 # <-- TYPO: non-existent version
```

Comment:
Not to nitpick, but "1.91" was a more likely example of a typo, cause it's easy to omit a dot :)

Reply:
but it's not as easy to notice it.. and IMO it's better to show the fact that if whatever you put there is "wrong" (non-existent) it won't work. spotting the difference between 1.9.1 vs 1.91 on a screen several meters far away during a training session is much harder.